### PR TITLE
Logging update added better wording around Kibana

### PR DESCRIPTION
**Pull request by mburke5678**
Improving wording in update section to reference the visualizations and index patterns are removed by update.

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge PTAL

File: logging/cluster-logging-upgrading.adoc [hunk 1]
```diff
@@ -9,13 +9,15 @@ toc::[]
 
 After updating the {product-title} cluster from 4.4 to 4.5, you can then update the Elasticsearch Operator and Cluster Logging Operator from 4.4 to 4.5.
 
-Cluster logging 4.5 introduces a new Elasticsearch version, Elasticsearch 6.8.1, and an enhanced security plug-in, Open Distro for Elasticsearch. The new Elasticsearch version introduces a new Elasticsearch data model. With the new data model, data is no longer indexed by type (infrastructure and application) and project. Because of these new features, you need to update your Kibana index patterns and visualizations to match the new indices after updating.
+Cluster logging 4.5 introduces a new Elasticsearch version, Elasticsearch 6.8.1, and an enhanced security plug-in, Open Distro for Elasticsearch. The new Elasticsearch version introduces a new Elasticsearch data model, where the Elasticsearch data is indexed by only by type: infrastructure, application, and audit. Previously, data was indexed by type (infrastructure and application) and project.
```
Comment (sichvoge) [inline logging/cluster-logging-upgrading.adoc:12]: "by only by type" ? :)

**Review by sichvoge — APPROVED** [review]
Added a small comment, which I believe might be a mistake only. Please make sure to look into it but otherwise LGTM.

**Comment by mburke5678** [comment]
Notes from meeting with Christian

[Christian Meeting Deprecate.txt](https://github.com/openshift/openshift-docs/files/4954564/Christian.Meeting.Deprecate.txt)

File: modules/cluster-logging-deploy-console.adoc [hunk 2]
```diff
@@ -67,9 +70,15 @@ scrapes the `openshift-operators-redhat` Namespace.
 +
 This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object. 
 You must select this option to ensure that cluster monitoring
-scrapes the `openshift-operators-redhat` Namespace.
+scrapes the `openshift-operators-redhat` namespace.
 
-.. Select an *Update Channel* and *Approval Strategy*.
+.. Select *4.5* as the *Update Channel*.
```
Comment (adellape) [inline modules/cluster-logging-deploy-console.adoc:75]: Same suggestion here re: unordered list.

**Review by adellape — CHANGES_REQUESTED** [review]
Some suggestions, but the `+` should be fixed at least.

**Pull request merged**